ecosystem/indexer/src/indexer/fetcher.rs

```diff
@@ -109,3 +120,23 @@ impl TransactionFetcher {
         }
     }
 }
+
+/// For mocking TransactionFetcher in tests
+#[async_trait::async_trait]
+pub trait TransactionFetcherTrait: Send + Sync {
```

Comment:
Just TransactionFetcher.

Reply:
Also from looking at the tests, it seems like some of these functions are just for the testing impl of this trait, is that right?

Reply:
yes!

Reply:
> Just TransactionFetcher

You mean remove Trait from TransactionFetcherTrait? It yells at me that TransactionFetcher has already been defined.